vanilla-installer/main.py

```diff
@@ -120,64 +167,99 @@ def install_fabric(
         text_update(f"Could not install Fabric: {ran}", widget, "error")
 
 
-def download_pack(widget=None):
-    text_update(f"Fetching Pack...", widget)
+def download_pack(widget, mc_dir: str) -> str:
+    """Downloads the files for FO.
 
-    pack_json = requests.get(
-        "https://api.github.com/repos/Fabulously-Optimized/fabulously-optimized/releases/latest"
-    ).json()
-    pack_file = ""
+    Downloads both the packwiz_install_bootstrap and the packwiz_installer jars.
```

Comment:
Why? bootstrap will download and launch installer by itself.

Reply:
Didn't know that actually, I'll just download bootstrap then

Reply:
Well that's what bootstrap is for, to download and keep updated packwiz-installer